Dockerfile

```diff
@@ -4,11 +4,14 @@ FROM python:3.12-slim-bullseye
 COPY --from=ghcr.io/astral-sh/uv:latest /uv /bin/uv
 
 # Copy the application into the container.
-COPY . /app
+COPY . /server
```

Comment:
Is there a best-practice name people use? `/repo` maybe?

Reply:
Usually /app 😄
But I've seen some images use /srv/_some-name_ which I suppose is [FHS-compliant](https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#srvDataForServicesProvidedBySystem) although that's not a big deal for Docker containers